AzureCalling/Controllers/CallViewController.swift

```diff
@@ -131,6 +135,15 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         cleanViewRendering()
     }
 
+    private func openAudioDeviceDrawer() {
+        let audioDeviceSelectionViewController = storyboard?.instantiateViewController(withIdentifier: "AudioDeviceSelectionViewController") as! AudioDeviceSelectionViewController
```

Comment:
Are we instantiating the viewcontroller from storyboard because the audio selection UI is defined in the Storyboard?

Reply:
It's to make IBOutlets instantiate.  If we just called AudioDeviceSelectionViewController(), the IBOutlet properties would be nil.